scripts/install_sodium.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
```

Comment:
For set command in BASH I'd like to always use this line unless there is a reason not too. I'd like to make this standard. Not a biggie but I think better.
set -euxo pipefail
This is a good overview of why. 
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
Maybe less is more but I think the use of set command is reasonable

Reply:
@jeremybusk this is brilliant! thx!

Reply:
alright, added!